Unreal/SystemLocalized/det/Glathriel1.det

```diff
@@ -12,11 +12,11 @@ LevelEntryText="Betrete das Dorf Glathriel"
 
 [TranslatorEvent0]
 ; EN: Message="I saw another spider in the basement today. They squeeze through the crack in the wall and take away whatever they want. I know I will have to close the crack, but how can I do it safely when any moment a spider could appear and threaten my life?"
-Message="Ich habe heute noch eine Spinne im Keller gesehen. Sie quetschen sich durch den Spalt in der Wand, und sie nehmen sich, was sie wollen. Ich weiß, ich werde den Spalt schließen müssen, aber wie kann ich das sicher machen, wenn jeden Moment eine Spinne kommen könnte und mir nach dem leben trachtet?"
+Message="Ich habe heute wieder eine Spinne im Keller gesehen. Sie quetschen sich durch den Spalt in der Wand und holen sich was sie wollen. Ich weiß dass ich den Spalt versiegeln muss, aber wie soll ich das tun, wenn jeden Augenblick eine Spinne auftauchen könnte um mein Leben zu bedrohen?"
```

Comment:
Message="Ich habe heute wieder eine Spinne im Keller gesehen. Sie quetschen sich durch den Spalt in der Wand und holen sich was sie wollen. Ich weiß dass ich den Spalt versiegeln muss, aber wie soll ich das tun, wenn jeden Augenblick eine Spinne auftauchen könnte, die mir mein Leben nehmen könnte?"